bower.json

```diff
@@ -4,8 +4,7 @@
     "homepage": "https://github.com/aspnet/jquery-validation-unobtrusive",
     "description": "Add-on to jQuery Validation to enable unobtrusive validation options in data-* attributes.",
     "main": [
-        "jquery.validate.unobtrusive.js",
-        "jquery.validate.unobtrusive.min.js"
+        "jquery.validate.unobtrusive.js"
     ],
     "ignore": "",
```

Comment:
Could you also add the necessary files to the `ignore` list?

BTW would you be interested in doing similar changes for this package too? https://github.com/aspnet/jquery-ajax-unobtrusive/blob/master/bower.json#L6